src/app/form-builder/form-builder-properties/form-builder-properties.component.html

```diff
@@ -1,12 +1,12 @@
-<h2>Properties</h2>
+<h3>Item Properties</h3>
 <div id="element-properties-controls-container"
      *ngIf="item$ | async ; let item">
   <div class="form-group">
 
-    <fieldset>
-      <legend>Base</legend>
+    <fieldset class="mb-4">
+      <legend>Machine Readable Name</legend>
```

Comment:
by "Base" i meant the __base__ item properties, which I think we should adapt to "Base Properties"